pkg/operator/whoown/members.csv

```diff
@@ -0,0 +1,4 @@
+Pillar Lead (M F),,12,,,,,,,C A,,,aos-group-b@redhat.com,,,,,,

```

Comment:
this is a sample of that CSV file... anonymized to some extend to about spam :-)